go.mod

```diff
@@ -88,183 +88,25 @@ require (
 	github.com/Azure/azure-amqp-common-go/v3 v3.1.0 // indirect
 	github.com/Azure/azure-sdk-for-go v52.6.0+incompatible // indirect
 	github.com/Azure/go-amqp v0.13.6 // indirect
-	github.com/Azure/go-autorest v14.2.0+incompatible // indirect
```

Comment:
Something is wrong with `go.mod`? Cleaning up so much shouldn't be right.

Reply:
true! could it be dependent on the go version? mine is go1.16.3 darwin/amd64
I can revert this file and push

Reply:
I have reverted this file but if I run make codegen it makes these changes in go.mod